sdks/builds/Makefile

```diff
@@ -11,31 +11,58 @@ RELEASE=$(if $(filter $(CONFIGURATION),release),1)
 
 lowercase=$(shell echo "$(1)" | tr '[:upper:]' '[:lower:]')
 
-CCACHE:=$(if $(DISABLE_CCACHE),,$(shell which ccache))
-NINJA:=$(shell which ninja)
-
 UNAME=$(shell uname)
 
+ifneq (,$(findstring CYGWIN,$(UNAME)))
+UNAME=Windows
+endif
+
+ifeq ($(UNAME),Linux)
+UNAME_WSL_CHECK=$(shell uname -a)
+ifneq (,$(findstring Microsoft,$(UNAME_WSL_CHECK)))
```

Comment:
Can this be different somehow? It breaks wasm on WSL. I don't know if it works, I'm trying it, but this definitely breaks it.